docker/dev/setup.txt

```diff
@@ -0,0 +1,19 @@
+Docker local development setup.
```

Comment:
I prefer putting the development documents under the root directory named `DEVELOPMENT.md` (and please use the markdown for any documentations)

Reply:
If it’s ok, I will move it in that file and covert it to markdown format.

Reply:
You can delete this file :)

Reply:
I forgot to delete 😅